recipes/nvcc/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "nvcc" %}
+{% set version = environ.get("CUDA_VER", "9.2") %}
+
+package:
+  name: "{{ name }}"
+
+build:
+  number: 0
+  skip: true  # [not linux]
+
+outputs:
+  - name: "{{ name }}_{{ target_platform }}"
+    version: "{{ version }}"
+    script: install_nvcc.sh
+    build:
+      script_env:
+        - CUDA_HOME
+      ignore_run_exports:
+        - libgcc-ng
+      run_exports:
+        strong:
+          - cudatoolkit {{ version }}|{{ version }}.*
```

Comment:
You need the `run_exports` of the C++ compiler here as well.

Reply:
Why? If the user is using the C++ compiler, that will be added by this anyways.

Reply:
But the runtimes like `libgcc-ng` and `libstdcxx-ng` will not be added. `run_exports` is not transitive.

Reply:
I don't think users should be relying on `nvcc` to add these things. If they are using a C or C++ compiler, they should list those requirements explicitly in their recipe.

Reply:
Then the C and C++ compiler should not be in run requirements. The way the compiler package is set up is that if the user tries to use a C++ compiler in the docker image, it will not be found if it is not an explicit requirement. Having the explicit requirement means that the `run_exports` for `libgcc-ng` is added.
But now, if the user adds `nvcc_linux-64`, they get the C++ compiler, but no `run_exports` for `libgcc-ng`.

Reply:
That makes sense. Have dropped the C and C++ compiler from `run`. They won't be installed now. The test has been updated to list C and C++ as requirements of the test.